soapclient/SforceEnterpriseClient.php

```diff
@@ -81,9 +76,9 @@ public function create($sObjects, $type) {
    * @return UpdateResult
    */
   public function update($sObjects, $type, $assignment_header = NULL, $mru_header = NULL) {
-
-    foreach ($sObjects as &$sObject) {
-
+	$arg = new stdClass;
+    	$arg->sObjects = [];

```

Comment:
Same remark, you could use the "classic" array syntax for backward compatibility

$arg->sObjects = array();